README.md

```diff
@@ -78,6 +76,46 @@ logger.end = log2gelf.end;
 * `release`: the version of your service (e.g. 1.0.0).
 * `_foo`: any underscore-prefixed custom option will be passed as is to the server.
 
+### Protocol-specific options
+
+#### `tcp` and `tls`
+
+* `reconnect`: Number of tcp reconnect attempts (default 0, 0 for none, -1 for infinite)
+* `wait`: Milliseconds to wait between reconnect attempts (default 1000)
+
+### Overriding connection and request options
+
+_**API change**: prior to version `2.3.0`, the `host`, `port` and `rejectUnauthorized` options were not overrideable by `protocolOptions`!_
```

Comment:
I also included an API change warning here with the assumption that the next version would be `2.3.0`.

Reply:
Agrred, it definitely diserves a version update.

I'm sorry it took me ages to check out your opened issue. I was reading it at the very moment you sent this PR.
Checking this right now.